static/canjs.js

```diff
@@ -444,6 +444,20 @@ function navigate(href, updateLocation) {
 			//root elements - use .filter; not .find
 			var $pathPrefixDiv = $content.filter("[path-prefix]");
 
+			// handle inline javascript code
+			// create a steal module on the fly
+			var scripts = $article[0].querySelectorAll('script[type="text/steal-module"]');
+			if (scripts && scripts.length > 0 ) {
+				$.each(scripts, function (i, script) {
+					if (typeof steal !== 'undefined') {
```

Comment:
Is this a case we have to worry about? If so, do we need to set up something to run when steal is defined, then define these modules?